includes/modules/shipping/usps.php

```diff
@@ -434,6 +434,21 @@ protected function checkConfiguration()
         return $this->enabled;
     }
 
+    // -----
+    // Storefront cart check.  Called from storefrontInitialization.  
+    // Will auto-disable the shipping if cart contents cannot be shipped
+    // by USPS
+    //
+    protected function checkCartContents() 
+    { 
+        $contents_ok = true;
+        $zco_notifier->notify('NOTIFY_SHIPPING_CHECK_CART', 'usps', $contents_ok); 
```

Comment:
Need to use `$this`, not `$zco_notifier` and change the notification to be named specifically for the USPS module (as previously requested, e.g. `NOTIFY_USPS_SHIPPING_CHECK_CART`).